src/components/Filter-time/FilterTime.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import { useState } from "react";
+import PropTypes from "prop-types";
+function FilterTime({time}) {
```

Comment:
change the branch name as it is the same as another branch, also format the code, and change the folder and component name to `FilterTime`